packages/valory/skills/abstract_round_abci/skill.yaml

```diff
@@ -17,9 +17,9 @@ fingerprint:
   handlers.py: bafybeidammqyvz2wvqirr3it4tfy7vbe5kxybwkz5o3y4ogo3wrbaqwcmy
   io_/__init__.py: bafybeiattfgecxl5y276etqe5jiuhqk3grcov6qky5cgokstmwhrbaodri
   io_/ipfs.py: bafybeic2lil23fk77wk3i6jrx47fdmc5wxr4dl2xbjjomgtfkz7yesz4eq
-  io_/load.py: bafybeiamag3kolsuv75skjgzfyosip3qqpdqzadet6niqej7x4wdostgdm
+  io_/load.py: bafybeifdki3qeq4aplu6chpozysmeqrt23lxhvcmlefsbesdrklh4wxu64
   io_/paths.py: bafybeicfno2l4vwtmjcm3rzpp6tqi3xlkof47pypf5teecad22d44u2ple
-  io_/store.py: bafybeiczvoewpvhlblwkewsx4b27urji73vgssrhxkkpqf4a6eesmqp62m
+  io_/store.py: bafybeihksp4ch67ae3lum5ou62mmctbl4xnodkiewdlbilmgy752m4pwc4
   models.py: bafybeifsna4hnkhgqegmhzh2uh62q6ikplvoqcsujnnvpol3wqozdhpcn4
   serializer.py: bafybeiezjdtp33nub7ual5ibt4vi7dj3ilmyw7wqq374ctub5oo7wgp7mm
```

Comment:
Also remove the dependencies below

Reply:
Done in 814d7f16736b67c64627c48a755f8bddd33f8209.